pkg/util/container/providers.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
```

Comment:
This also needs a `docker` build tag, I think. Also may/may not need to be stubbed if the `docker` tag isn't included?

Reply:
you're right 👍 https://github.com/DataDog/datadog-agent/pull/1034 will fix this